windtunnel/post_processing.py

```diff
@@ -88,3 +88,24 @@ def get_force_coefficients(sim_output_path: str):
         zip(coefficients_labels, map(float, coefficients[1:])))
 
     return coefficients_dict
+
+
+def get_streamlines(
+        sim_output_path: str,
+        object_mesh: pv.PolyData,
```

Comment:
should be domain_mesh

Reply:
This had already been solved in another commit (directly to main) https://github.com/inductiva/wind-tunnel/commit/8cf3d4cccb377fad7b4865004f73805b47b6dc80